### DISCUSSION TITLE
page inspector

### DISCUSSION THREAD

**Discussion by lsag** [zim-desktop-wiki/zim-desktop-wiki#2628] [Q&A] [answered]
Hello. Is the a way to inspect css and html being used in a zim-wiki page, like in chrome? 
This is my System:
- Ubuntu Release 22.04.4 LTS (Jammy Jellyfish) 64-bit
- MATE 1.26.0

Thanks

**Comment by jaap-karssenberg** [comment, marked as answer]
No. We don't use html or css in the application.

The edit bar does have an info field on the formatting that is applied at
the cursor position.

Regards

Jaap

Op do 20 jun. 2024 19:33 schreef lsag ***@***.***>:
